diff --git a/node.js b/node.js
index 79438b7a92be138777df20ab9db21a08d6178be4..3a1fdc32905ac53ffc56cd20c8d9d4fd4c8caf3b 100644
--- a/node.js
+++ b/node.js
@@ -1,9 +1,9 @@
 module.exports = {
   extends: ['./shared/core.js', './shared/typescript.js', './shared/prettier.js'],
-  plugins: ['node'],
+  plugins: ['n'],
   env: { node: true },
   rules: {
     'no-buffer-constructor': 'warn',
-    'node/no-path-concat': 'warn',
+    'n/no-path-concat': 'warn',
   },
 };
diff --git a/shared/core.js b/shared/core.js
index f3e09f926b287f8dea3f2a73bd1f2f9dd9d7fd85..d417e43437d3f2e9c1a81d4afcccc6a8e88c08c3 100644
--- a/shared/core.js
+++ b/shared/core.js
@@ -14,7 +14,7 @@ module.exports = {
     module: false,
     require: false,
   },
-  plugins: ['import', 'node'],
+  plugins: ['import', 'n'],
   rules: {
     'array-bracket-spacing': ['warn', 'never'],
     'arrow-spacing': ['warn', { before: true, after: true }],
@@ -156,8 +156,8 @@ module.exports = {
       },
     ],
 
-    'node/handle-callback-err': ['warn', '^(e|err|error|.+Error)$'],
-    'node/no-new-require': 'warn',
+    'n/handle-callback-err': ['warn', '^(e|err|error|.+Error)$'],
+    'n/no-new-require': 'warn',
   },
   settings: {
     'import/extensions': jsExtensions,
